usr.bin/lsvfs/lsvfs.c

```diff
@@ -40,10 +45,29 @@ main(int argc, char **argv)
 {
 	struct xvfsconf vfc, *xvfsp;
 	size_t buflen;
-	int cnt, i, rv = 0;
+	int cnt, rv = 0;
+	cap_channel_t *capcas, *capsysctl;
+	cap_sysctl_limit_t *limit;
 
 	argc--, argv++;
 
+	if ((capcas = cap_init()) == NULL)
+		err(EXIT_FAILURE, "unable to contact Casper");
+	if ((capsysctl = cap_service_open(capcas, "system.sysctl")) == NULL)
+		err(EXIT_FAILURE, "unable to open system.sysctl servic");
+	cap_close(capcas);
+
+	if ((limit = cap_sysctl_limit_init(capsysctl)) == NULL)
```

Comment:
We cannot check here for a NULL. As the non-casper version of the cap_sysctl_limit_init will return name as well. I guess we need to update man page for this (the return value section). The idea is that you ignore errors from `cap_sysctl_limit_init` ect, and verify only `cap_sysctl_limit`. See examples section of man page.

Reply:
Alright, I'll take care of it now, and will also take care of the man page later today.

Reply:
I guess we/I need to update the `cap_sysctl_limit_name` of the real function to ignore NULL limit.

Reply:
Good idea, makes sense. Alright.

Reply:
The latest commit should meet your call now. Let me know if any other change's required as well.